test/flows/vat_payment_deadlines_flow_test.rb

```diff
@@ -0,0 +1,85 @@
+require "test_helper"
+require "support/flow_test_helper"
+
+class VatPaymentDeadlinesFlowTest < ActiveSupport::TestCase
+  include FlowTestHelper
+
+  setup do
+    WebMock.stub_request(:get, WorkingDays::BANK_HOLIDAYS_URL)
+      .to_return(body: File.open(fixture_file("bank_holidays.json")))
+    testing_flow VatPaymentDeadlinesFlow
+  end
+
+  should "render a start page" do
+    assert_rendered_start_page
+  end
+
+  context "question: when_does_your_vat_accounting_period_end?" do
+    setup { testing_node :when_does_your_vat_accounting_period_end? }
+
+    should "render the question" do
+      assert_rendered_question
+    end
+
+    context "next_node" do
+      should "have a next node of how_do_you_want_to_pay? for a date of '2013-04-30' response" do
```

Comment:
```suggestion
      should "have a next node of how_do_you_want_to_pay? for a date of valid response" do
```

Specifying the response in the description creates the misleading impression that that response influences the next node.